src/overlays/actors/ovl_En_Aob_01/z_en_aob_01.c

```diff
@@ -465,24 +465,29 @@ void func_809C1D64(EnAob01* this, GlobalContext* globalCtx) {
 }
 
 void func_809C1EC8(EnAob01* this, GlobalContext* globalCtx) {
-    static u16 D_809C392C[] = { 4000, 4, 1, 3, 6000, 4, 1, 6, 4000, 4, 1, 3, 6000, 4, 1, 6 };
+    static u16 sTurnOptions[][4] = {
```

Comment:
You might as well put both dimensions in... frankly I'm even considering a typedef at this point... could it be a struct, an array of structs?

Reply:
Structs do work, just wasn't sure if we wanted to add structs just for this one function. Personally if we did structs I would do something like 

```c
typedef struct ComputeTurnToPointOptions {
    /* 0x0 */ u16 rotMax; // in binary angles
    /* 0x2 */ u16 slowness;
    /* 0x4 */ u16 stepMin; // in degrees
    /* 0x6 */ u16 stepMax; // in degrees
} ComputeTurnToPointOptions; // size = 0x8

typedef struct TurnToPointOptions {
    /* 0x00 */ ComputeTurnToPointOptions headRotX;
    /* 0x08 */ ComputeTurnToPointOptions headRotY;
    /* 0x10 */ ComputeTurnToPointOptions torsoRotX;
    /* 0x18 */ ComputeTurnToPointOptions torsoRotY; // size = 0x20
} TurnToPointOptions;
```

Reply:
That certainly seems clearer, although feels like the names could be improved.

Reply:
Suggestions?

Reply:
Certainly the function name should be involved to an extent. But I don't like the names for the "rows". Hmm, how about TurnOptions and TurnOptionsSet or something?

Reply:
So just to clear like
```c
typedef struct TurnOptions {
    /* 0x0 */ u16 rotMax; // in binary angles
    /* 0x2 */ u16 slowness;
    /* 0x4 */ u16 stepMin; // in degrees
    /* 0x6 */ u16 stepMax; // in degrees
} TurnOptions; // size = 0x8

typedef struct TurnOptionsSet {
    /* 0x00 */ ComputeTurnOptions headRotX;
    /* 0x08 */ ComputeTurnOptions headRotY;
    /* 0x10 */ ComputeTurnOptions torsoRotX;
    /* 0x18 */ ComputeTurnOptions torsoRotY; 
} TurnOptionsSet; // size = 0x20
```
?

Reply:
Yeah, although just "binary angle" and "degrees" would do, probably.